tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -540,6 +541,7 @@ static void test_cis_link_loss_in_disabling_state(struct ascs_test_suite_fixture
 
 	/* Expected to notify the upper layers */
 	expect_bt_bap_stream_ops_qos_set_called_once(stream);
+	expect_bt_bap_stream_ops_disabled_not_called();
```

Comment:
Shouldn't this be conditional on 'streaming' ?
If streaming is set to true, then we have reached the streaming state and should transition to disable, which would invoke the disabled cb ?

Reply:
There is a previously call to transit to the disabled state, so we are not in the streaming state anymore